### ISSUE TITLE
Sync fails

### ISSUE THREAD

**Issue by akwala** [wallabag/android-app#371]
Sync fails, reporting connection error, but the connection test succeeds. This has been happening since installing the app. Data hosted at framabag.org. 

Wallabag app for Android v. 1.11.0 (29)
Android 6.0.1 (CM13 Hammerhead)

**Comment by akwala** [comment]
The error:
https://drive.google.com/file/d/0B-Q1-r3CGeBBM1Z3NnpGdXZTZUE/view?usp=drivesdk

Message from connection test done immediately after the above error:
https://drive.google.com/file/d/0B-Q1-r3CGeBBM1Z3NnpGdXZTZUE/view?usp=drivesdk

I tried filtering the Main log using several keywords (class names, class path elements, etc.) but did not see anything related to the error. If you have suggestions I can try them. 

Note: I set the credentials anew yesterday, which did not report any errors.

**Comment by akwala** [comment]
Correction:

Message from connection test done immediately after the above error:
https://drive.google.com/file/d/0B-Q1-r3CGeBBMGlpZC03OGdxcjA/view?usp=drivesdk

**Comment by di72nn** [comment]
The "triangle with an exclamation mark" icon in the status bar on the first screenshot is likely to be wallabag's error notification. If so, can you check what does it say (in your case it is likely to be either "Incorrect credentials" or "Incorrect configuration")?

Try to filter logcat messages with `adb shell 'logcat --pid=$(pidof -s fr.gaulupeau.apps.InThePoche)'`, or if the command doesn't work:
 1. Get the process ID:
  `adb shell pidof fr.gaulupeau.apps.InThePoche`
 2. Filter logcat:
  `adb shell 'logcat | grep <the process id from the previous step>'`

**Comment by akwala** [comment]
The notification said "incorrect configuration."

**Comment by akwala** [comment]
@di72nn Here's what I got from logcat using your second method (after launching the app and seeing the "Problem detected" box):

```
12-16 18:32:28.617   826  1825 I ActivityManager: Start proc 28418:fr.gaulupeau.apps.InThePoche/u0a191 for activity fr.gaulupeau.apps.InThePoche/fr.gaulupeau.apps.Poche.ui.ArticlesListActivity
12-16 18:32:28.747 28418 28418 W System  : ClassLoader referenced unknown path: /data/app/fr.gaulupeau.apps.InThePoche-1/lib/arm
12-16 18:32:28.774 28418 28418 I LoadedApk: No resource references to update in package common
12-16 18:32:28.775 28418 28418 I LoadedApk: No resource references to update in package ideadesigns.theme.britzer
12-16 18:32:28.775 28418 28418 I LoadedApk: No resource references to update in package ind.aang.midnight.blue
12-16 18:32:28.775 28418 28418 I LoadedApk: No resource references to update in package ideadesigns.theme.britzer
12-16 18:32:28.941 28418 28418 W art     : Before Android 4.1, method android.graphics.PorterDuffColorFilter android.support.graphics.drawable.VectorDrawableCompat.updateTintFilter(android.graphics.PorterDuffColorFilter, android.content.res.ColorStateList, android.graphics.PorterDuff$Mode) would have incorrectly overridden the package-private method in android.graphics.drawable.Drawable
12-16 18:32:29.047 28418 28418 D ArticlesListActivity: checkLists() started
12-16 18:32:29.047 28418 28418 D ArticlesListActivity: checkLists() finished
12-16 18:32:29.080 28418 28452 D OpenGLRenderer: Use EGL_SWAP_BEHAVIOR_PRESERVED: true
12-16 18:32:29.135 28418 28452 I Adreno-EGL: <qeglDrvAPI_eglInitialize:379>: QUALCOMM Build: 10/21/15, 369a2ea, I96aee987eb
12-16 18:32:29.136 28418 28452 I OpenGLRenderer: Initialized EGL, version 1.4
12-16 18:32:29.166 28418 28418 D ArtListPagerAdapter: getItem 1
12-16 18:32:29.166 28418 28418 D ArtListPagerAdapter: getFragment 1
12-16 18:32:29.166 28418 28418 D ArtListPagerAdapter: creating new instance
12-16 18:32:29.170 28418 28418 D ArtListPagerAdapter: getItem 0
12-16 18:32:29.170 28418 28418 D ArtListPagerAdapter: getFragment 0
12-16 18:32:29.170 28418 28418 D ArtListPagerAdapter: creating new instance
12-16 18:32:29.170 28418 28418 D ArtListPagerAdapter: getItem 2
12-16 18:32:29.170 28418 28418 D ArtListPagerAdapter: getFragment 2
12-16 18:32:29.170 28418 28418 D ArtListPagerAdapter: creating new instance
12-16 18:32:29.170 28418 28418 D ArticlesListFragment: Fragment 0 onAttach()
12-16 18:32:29.172 28418 28418 D DbConnection: creating new db session
12-16 18:32:29.188 28418 28418 D DbConnection: using existing db session
12-16 18:32:29.189 28418 28418 D ArticlesListFragment: Fragment 0 onAttach()
12-16 18:32:29.189 28418 28418 D DbConnection: using existing db session
12-16 18:32:29.189 28418 28418 D ArticlesListFragment: Fragment 0 onAttach()
12-16 18:32:29.189 28418 28418 D DbConnection: using existing db session
12-16 18:32:29.210 28418 28418 D ArticlesListFragment: Fragment 0 onResume()
12-16 18:32:29.215 28418 28418 D ArticlesListFragment: Fragment 0 onShow()
12-16 18:32:29.218 28418 28418 D ArticlesListFragment: Fragment 1 onResume()
12-16 18:32:29.219 28418 28418 D ArticlesListFragment: Fragment 1 onShow()
12-16 18:32:29.219 28418 28418 D ArticlesListFragment: Fragment 2 onResume()
12-16 18:32:29.219 28418 28418 D ArticlesListFragment: Fragment 2 onShow()
12-16 18:33:28.515 28418 28418 D ArticlesListFragment: Fragment 0 onPause()
12-16 18:33:28.516 28418 28418 D ArticlesListFragment: Fragment 1 onPause()
12-16 18:33:28.516 28418 28418 D ArticlesListFragment: Fragment 2 onPause()

```

**Comment by di72nn** [comment]
The "Incorrect configuration" message is shown only if the server URL is not correct (according to standards) or a response from server is not HTTP OK.
Did you use the connection wizard to configure the app?

Logs from the moment right before starting "Update feed" operation to the moment when the notification with an exclamation mark appears would me more suitable.

**Comment by rezad1393** [comment]
i have the same error (vague error) about failing to sync on v2.wallabag.org
ob both phone and tablet.

**Comment by di72nn** [comment]
@rezad1393 this seems to be a different issue. Please open a new issue if you want to get help with it.

**Comment by rezad1393** [comment]
ok

**Comment by akwala** [comment]
@di72nn Yes, I used the connection wizard. The server URL is of the form https://framabag.org/u/username. 

Here's what happens when I launch the app:
1. The "Problem detected" box appears immediately. 
2. I tap OK in the box to test the connection. 
3. Message "Testing connection..." pops up, then disappears. 
4. Message "Checking feeds..." pops up, sync animation appears and sync symbol appears in status bar - all for only a second or so. 
5. "Parameters are fine" pops up and the triangle with exclamation mark appears in the status bar with the "configuration error" notification. 

The log excerpt I sent was after the above steps. IOW, the log was excerpted following the appearance of the triangle with exclamation mark.

**Comment by di72nn** [comment]
Ok, these are two separate processes collided: the connection test and feed update (started because db is empty). That is not the problem, though.

Try the following:
 1. Launch the app.
 1. Press "Not now" in the "Problem detected" dialog.
 1. Wait for the auto-started update operation to finish with an error (in the notification bar; you may throw the notification away).
 1. Start logs gathering.
 1. Run "Update all feeds", wait for it to finish with an error.
 1. Stop logs gathering. Copy the logs here.

**Comment by akwala** [comment]
Followed your steps. 

This time, the update (step 3) did not seem to auto-start and neither the sync symbol nor the error symbol+notification appeared.

Upon tapping the sync/update symbol (step 5), "Please configure the app before syncing!" popped up at the bottom. AFAIKT the config is fine.

Log excerpt:
```
$ adb shell 'logcat | grep 4846'              
12-18 14:58:23.847   826  1816 I ActivityManager: Start proc 4846:fr.gaulupeau.apps.InThePoche/u0a191 for activity fr.gaulupeau.apps.InThePoche/fr.gaulupeau.apps.Poche.ui.ArticlesListActivity
12-18 14:58:23.946  4846  4846 W System  : ClassLoader referenced unknown path: /data/app/fr.gaulupeau.apps.InThePoche-1/lib/arm
12-18 14:58:23.952  4846  4846 I LoadedApk: No resource references to update in package common
12-18 14:58:23.953  4846  4846 I LoadedApk: No resource references to update in package ideadesigns.theme.britzer
12-18 14:58:23.953  4846  4846 I LoadedApk: No resource references to update in package ind.aang.midnight.blue
12-18 14:58:23.953  4846  4846 I LoadedApk: No resource references to update in package ideadesigns.theme.britzer
12-18 14:58:24.027  4846  4846 W art     : Before Android 4.1, method android.graphics.PorterDuffColorFilter android.support.graphics.drawable.VectorDrawableCompat.updateTintFilter(android.graphics.PorterDuffColorFilter, android.content.res.ColorStateList, android.graphics.PorterDuff$Mode) would have incorrectly overridden the package-private method in android.graphics.drawable.Drawable
12-18 14:58:24.092  4846  4846 D ArticlesListActivity: checkLists() started
12-18 14:58:24.092  4846  4846 D ArticlesListActivity: checkLists() finished
12-18 14:58:24.148  4846  4878 D OpenGLRenderer: Use EGL_SWAP_BEHAVIOR_PRESERVED: true
12-18 14:58:24.221  4846  4878 I Adreno-EGL: <qeglDrvAPI_eglInitialize:379>: QUALCOMM Build: 10/21/15, 369a2ea, I96aee987eb
12-18 14:58:24.224  4846  4878 I OpenGLRenderer: Initialized EGL, version 1.4
12-18 14:58:24.247  4846  4846 D ArtListPagerAdapter: getItem 1
12-18 14:58:24.247  4846  4846 D ArtListPagerAdapter: getFragment 1
12-18 14:58:24.247  4846  4846 D ArtListPagerAdapter: creating new instance
12-18 14:58:24.249  4846  4846 D ArtListPagerAdapter: getItem 0
12-18 14:58:24.249  4846  4846 D ArtListPagerAdapter: getFragment 0
12-18 14:58:24.249  4846  4846 D ArtListPagerAdapter: creating new instance
12-18 14:58:24.249  4846  4846 D ArtListPagerAdapter: getItem 2
12-18 14:58:24.249  4846  4846 D ArtListPagerAdapter: getFragment 2
12-18 14:58:24.249  4846  4846 D ArtListPagerAdapter: creating new instance
12-18 14:58:24.249  4846  4846 D ArticlesListFragment: Fragment 0 onAttach()
12-18 14:58:24.249  4846  4846 D DbConnection: creating new db session
12-18 14:58:24.260  4846  4846 D DbConnection: using existing db session
12-18 14:58:24.261  4846  4846 D ArticlesListFragment: Fragment 0 onAttach()
12-18 14:58:24.261  4846  4846 D DbConnection: using existing db session
12-18 14:58:24.261  4846  4846 D ArticlesListFragment: Fragment 0 onAttach()
12-18 14:58:24.261  4846  4846 D DbConnection: using existing db session
12-18 14:58:24.283  4846  4846 D ArticlesListFragment: Fragment 0 onResume()
12-18 14:58:24.287  4846  4846 D ArticlesListFragment: Fragment 0 onShow()
12-18 14:58:24.290  4846  4846 D ArticlesListFragment: Fragment 1 onResume()
12-18 14:58:24.291  4846  4846 D ArticlesListFragment: Fragment 1 onShow()
12-18 14:58:24.291  4846  4846 D ArticlesListFragment: Fragment 2 onResume()
12-18 14:58:24.292  4846  4846 D ArticlesListFragment: Fragment 2 onShow()
12-18 14:58:54.522  4846  4878 V RenderScript: 0x9ee78000 Launching thread(s), CPUs 4
12-18 14:59:12.707  4846  4846 D ArticlesListFragment: Fragment 0 onPause()
12-18 14:59:12.728  4846  4846 D ArticlesListFragment: Fragment 1 onPause()
12-18 14:59:12.728  4846  4846 D ArticlesListFragment: Fragment 2 onPause()
12-18 14:59:15.839   826  1013 I ActivityManager: Killing 4846:fr.gaulupeau.apps.InThePoche/u0a191 (adj 9): remove task
```

**Comment by di72nn** [comment]
"Please configure the app before syncing!" means that the sync didn't start, so the latest logs excerpt is useless.

The key idea now is to get logs with a failed attempt to sync.
Try to open the app, start gathering logs, and then do whatever it takes to start sync. If the app suggest to test settings - accept it. If it doesn't suggest anything, but the sync doesn't start (the "Please configure the app before syncing!" message is shown instead), go to settings and try to use "Autofill" and then go back to the main screen to sync. The more logs you get the better.

**Comment by akwala** [comment]
> If the app suggest to test settings - accept it.

This case is the same as that corresponding to the first log excerpt I posted.

The following log excerpt (scrubbed) corresponds to the following steps:
1. Launch app -- "Problem detected" box appears.
2. Tap "NOT NOW" in box.
3. Go to Settings->Connection and tap Autofill -- messages appear in a sequence, the last is "Parameters filled"
4. Shutdown the app.

No error message/symbol/notification appeared.

```
$ adb shell 'logcat | grep 28049'             
12-18 16:07:07.635   826  1854 I ActivityManager: Start proc 28049:fr.gaulupeau.apps.InThePoche/u0a191 for activity fr.gaulupeau.apps.InThePoche/fr.gaulupeau.apps.Poche.ui.ArticlesListActivity
12-18 16:07:07.762 28049 28049 W System  : ClassLoader referenced unknown path: /data/app/fr.gaulupeau.apps.InThePoche-1/lib/arm
12-18 16:07:07.799 28049 28049 I LoadedApk: No resource references to update in package common
12-18 16:07:07.799 28049 28049 I LoadedApk: No resource references to update in package ideadesigns.theme.britzer
12-18 16:07:07.799 28049 28049 I LoadedApk: No resource references to update in package ind.aang.midnight.blue
12-18 16:07:07.799 28049 28049 I LoadedApk: No resource references to update in package ideadesigns.theme.britzer
12-18 16:07:07.970 28049 28049 W art     : Before Android 4.1, method android.graphics.PorterDuffColorFilter android.support.graphics.drawable.VectorDrawableCompat.updateTintFilter(android.graphics.PorterDuffColorFilter, android.content.res.ColorStateList, android.graphics.PorterDuff$Mode) would have incorrectly overridden the package-private method in android.graphics.drawable.Drawable
12-18 16:07:08.090 28049 28049 D ArticlesListActivity: checkLists() started
12-18 16:07:08.090 28049 28049 D ArticlesListActivity: checkLists() finished
12-18 16:07:08.127 28049 28079 D OpenGLRenderer: Use EGL_SWAP_BEHAVIOR_PRESERVED: true
12-18 16:07:08.188 28049 28079 I Adreno-EGL: <qeglDrvAPI_eglInitialize:379>: QUALCOMM Build: 10/21/15, 369a2ea, I96aee987eb
12-18 16:07:08.191 28049 28079 I OpenGLRenderer: Initialized EGL, version 1.4
12-18 16:07:08.207 28049 28049 D ArtListPagerAdapter: getItem 1
12-18 16:07:08.207 28049 28049 D ArtListPagerAdapter: getFragment 1
12-18 16:07:08.207 28049 28049 D ArtListPagerAdapter: creating new instance
12-18 16:07:08.212 28049 28049 D ArtListPagerAdapter: getItem 0
12-18 16:07:08.212 28049 28049 D ArtListPagerAdapter: getFragment 0
12-18 16:07:08.212 28049 28049 D ArtListPagerAdapter: creating new instance
12-18 16:07:08.213 28049 28049 D ArtListPagerAdapter: getItem 2
12-18 16:07:08.213 28049 28049 D ArtListPagerAdapter: getFragment 2
12-18 16:07:08.213 28049 28049 D ArtListPagerAdapter: creating new instance
12-18 16:07:08.213 28049 28049 D ArticlesListFragment: Fragment 0 onAttach()
12-18 16:07:08.214 28049 28049 D DbConnection: creating new db session
12-18 16:07:08.225 28049 28049 D DbConnection: using existing db session
12-18 16:07:08.225 28049 28049 D ArticlesListFragment: Fragment 0 onAttach()
12-18 16:07:08.225 28049 28049 D DbConnection: using existing db session
12-18 16:07:08.225 28049 28049 D ArticlesListFragment: Fragment 0 onAttach()
12-18 16:07:08.225 28049 28049 D DbConnection: using existing db session
12-18 16:07:08.245 28049 28049 D ArticlesListFragment: Fragment 0 onResume()
12-18 16:07:08.249 28049 28049 D ArticlesListFragment: Fragment 0 onShow()
12-18 16:07:08.252 28049 28049 D ArticlesListFragment: Fragment 1 onResume()
12-18 16:07:08.253 28049 28049 D ArticlesListFragment: Fragment 1 onShow()
12-18 16:07:08.253 28049 28049 D ArticlesListFragment: Fragment 2 onResume()
12-18 16:07:08.253 28049 28049 D ArticlesListFragment: Fragment 2 onShow()
12-18 16:07:13.042 28049 28049 I ListPopupWindow: Could not find method setEpicenterBounds(Rect) on PopupWindow. Oh well.
12-18 16:07:13.060 28049 28049 W art     : Before Android 4.1, method int android.support.v7.widget.ListViewCompat.lookForSelectablePosition(int, boolean) would have incorrectly overridden the package-private method in android.widget.ListView
12-18 16:07:15.199 28049 28049 D ArticlesListFragment: Fragment 0 onPause()
12-18 16:07:15.199 28049 28049 D ArticlesListFragment: Fragment 1 onPause()
12-18 16:07:15.199 28049 28049 D ArticlesListFragment: Fragment 2 onPause()
12-18 16:07:15.290 28049 28049 D SettingsFragment: onStart() started
12-18 16:07:15.291 28049 28049 D SettingsFragment: resetChanges() started
12-18 16:07:15.436 28049 28079 D OpenGLRenderer: endAllStagingAnimators on 0x9de4f000 (MenuPopupWindow$MenuDropDownListView) with handle 0x9e2ff000
12-18 16:08:31.257 28049 28049 W InputEventReceiver: Attempted to finish an input event but the input event receiver has already been disposed.
12-18 16:08:37.034 28049 28548 D TestConnectionTask: Detecting redirection for: https://framabag.org/u/username
12-18 16:08:37.730 28049 28548 D TestConnectionTask: URLs: 
12-18 16:08:37.730 28049 28548 D TestConnectionTask: https://framabag.org/u/username
12-18 16:08:37.730 28049 28548 I TestConnectionTask: Testing https://framabag.org/u/username
12-18 16:08:37.731 28049 28548 D WallabagService: getBodyFromHttpResponse() url=https://framabag.org/u/username/api/version
12-18 16:08:38.216 28049 28548 D WallabagService: getBodyFromHttpResponse() response.code is not OK: 404
12-18 16:08:38.216 28049 28548 D WallabagService: isWallabagVersion2() did not find Wallabag v2
12-18 16:08:38.216 28049 28548 D WallabagService: getBodyFromHttpResponse() url=https://framabag.org/u/username/?view=about
12-18 16:08:38.682 28049 28548 D WallabagService: isWallabagVersion1() found Wallabag v1
12-18 16:08:39.225 28049 28548 D TestConnectionTask: Connection test result: OK
12-18 16:08:39.257 28049 28049 D ConfigurationTestHelper: connectionTestOnSuccess() new url: https://framabag.org/u/username
12-18 16:08:39.266 28049 28563 D WallabagServiceEndpoint: getCredentials(): configPath=/?view=config credentialsPattern="\?feed&amp;type=home&amp;user_id=(\d+)&amp;token=([a-zA-Z0-9]+)"
12-18 16:08:39.266 28049 28563 D WallabagServiceEndpoint: getConfigRequest() url: https://framabag.org/u/username/?view=config
12-18 16:08:39.266 28049 28563 D WallabagServiceEndpointV1: executeRequestForResult(url: https://framabag.org/u/username/?view=config, checkResponse: true, autoRelogin: true) started
12-18 16:08:39.909 28049 28563 D WallabagServiceEndpointV1: executeRequestForResult() got response
12-18 16:08:39.912 28049 28563 D WallabagServiceEndpointV1: executeRequestForResult() response is login page
12-18 16:08:39.912 28049 28563 D WallabagServiceEndpointV1: executeRequestForResult() trying to re-login
12-18 16:08:40.214 28049 28563 D WallabagServiceEndpointV1: executeRequestForResult() re-login response is OK; re-executing request
12-18 16:08:40.340 28049 28563 D WallabagServiceEndpointV1: executeRequestForResult() finished; returning page body
12-18 16:08:40.341 28049 28563 D WallabagServiceEndpoint: credentials=FeedsCredentials{userID='9999', token='dpokt'}
12-18 16:08:40.939 28049 28573 I TestFeedsTask: Result: OK
12-18 16:08:40.959 28049 28049 D SettingsFragment: onConfigurationTestSuccess(https://framabag.org/u/username, 9999)
12-18 16:08:40.960 28049 28049 D SettingsFragment: onSharedPreferenceChanged(internal.configurationIsOk)
12-18 16:08:40.960 28049 28049 D SettingsFragment: onSharedPreferenceChanged(internal.configurationErrorShown)
12-18 16:08:41.125 28049 28079 V RenderScript: 0x9bf15000 Launching thread(s), CPUs 4
12-18 16:08:53.709 28049 28049 D SettingsFragment: onStop() started
12-18 16:08:53.710 28049 28049 D SettingsFragment: applyChanges() started
12-18 16:08:55.906 28049 28049 D ArticlesListFragment: Fragment 0 onDetach()
12-18 16:08:55.906 28049 28049 D ArticlesListFragment: Fragment 1 onDetach()
12-18 16:08:55.906 28049 28049 D ArticlesListFragment: Fragment 2 onDetach()
12-18 16:08:55.908   826  1667 I ActivityManager: Killing 28049:fr.gaulupeau.apps.InThePoche/u0a191 (adj 9): remove task

```

**Comment by di72nn** [comment]
Well, I said to do anything **in order to run sync**. And to get the logs of a failed sync attempt. It is okay if logs contain other activities, but it is absolutely crucial for them to contain a failed sync attempt.
So, instead of your fourth step you should have started the sync.

I'm sorry that it takes so much effort, but I have no better idea how to handle the situation.

**Comment by akwala** [comment]
@di72nn Apologies. I thought the sync was attempted since "Checking feeds..." appeared as a result of Autofill. OK, here you go... same steps as the last time but with the following steps after Autofill completed:

4. Return to the home/dashboard screen (with the Favorites, Unread and Archived tabs) -- this apparently initiated a sync (animation appeared) and the error symbol+notification appeared.
5. Tap the update/sync button -- "Please configure the app before syncing!" popped up at the bottom.

I see that IncorrectConfigurationException is thrown.

```
$ adb shell 'logcat | grep 5202'
12-19 14:06:12.219   826  1855 I ActivityManager: Start proc 5202:fr.gaulupeau.apps.InThePoche/u0a191 for activity fr.gaulupeau.apps.InThePoche/fr.gaulupeau.apps.Poche.ui.ArticlesListActivity
12-19 14:06:12.303  5202  5202 W System  : ClassLoader referenced unknown path: /data/app/fr.gaulupeau.apps.InThePoche-1/lib/arm
12-19 14:06:12.305  5202  5202 I LoadedApk: No resource references to update in package common
12-19 14:06:12.305  5202  5202 I LoadedApk: No resource references to update in package ideadesigns.theme.britzer
12-19 14:06:12.305  5202  5202 I LoadedApk: No resource references to update in package ind.aang.midnight.blue
12-19 14:06:12.306  5202  5202 I LoadedApk: No resource references to update in package ideadesigns.theme.britzer
12-19 14:06:12.371  5202  5202 W art     : Before Android 4.1, method android.graphics.PorterDuffColorFilter android.support.graphics.drawable.VectorDrawableCompat.updateTintFilter(android.graphics.PorterDuffColorFilter, android.content.res.ColorStateList, android.graphics.PorterDuff$Mode) would have incorrectly overridden the package-private method in android.graphics.drawable.Drawable
12-19 14:06:12.432  5202  5202 D ArticlesListActivity: checkLists() started
12-19 14:06:12.432  5202  5202 D ArticlesListActivity: checkLists() finished
12-19 14:06:12.462  5202  5228 D OpenGLRenderer: Use EGL_SWAP_BEHAVIOR_PRESERVED: true
12-19 14:06:12.517  5202  5228 I Adreno-EGL: <qeglDrvAPI_eglInitialize:379>: QUALCOMM Build: 10/21/15, 369a2ea, I96aee987eb
12-19 14:06:12.519  5202  5228 I OpenGLRenderer: Initialized EGL, version 1.4
12-19 14:06:12.540  5202  5202 D ArtListPagerAdapter: getItem 1
12-19 14:06:12.540  5202  5202 D ArtListPagerAdapter: getFragment 1
12-19 14:06:12.540  5202  5202 D ArtListPagerAdapter: creating new instance
12-19 14:06:12.541  5202  5202 D ArtListPagerAdapter: getItem 0
12-19 14:06:12.541  5202  5202 D ArtListPagerAdapter: getFragment 0
12-19 14:06:12.541  5202  5202 D ArtListPagerAdapter: creating new instance
12-19 14:06:12.542  5202  5202 D ArtListPagerAdapter: getItem 2
12-19 14:06:12.542  5202  5202 D ArtListPagerAdapter: getFragment 2
12-19 14:06:12.542  5202  5202 D ArtListPagerAdapter: creating new instance
12-19 14:06:12.542  5202  5202 D ArticlesListFragment: Fragment 0 onAttach()
12-19 14:06:12.542  5202  5202 D DbConnection: creating new db session
12-19 14:06:12.552  5202  5202 D DbConnection: using existing db session
12-19 14:06:12.553  5202  5202 D ArticlesListFragment: Fragment 0 onAttach()
12-19 14:06:12.553  5202  5202 D DbConnection: using existing db session
12-19 14:06:12.553  5202  5202 D ArticlesListFragment: Fragment 0 onAttach()
12-19 14:06:12.553  5202  5202 D DbConnection: using existing db session
12-19 14:06:12.571  5202  5202 D ArticlesListFragment: Fragment 0 onResume()
12-19 14:06:12.574  5202  5202 D ArticlesListFragment: Fragment 0 onShow()
12-19 14:06:12.576  5202  5202 D ArticlesListFragment: Fragment 1 onResume()
12-19 14:06:12.577  5202  5202 D ArticlesListFragment: Fragment 1 onShow()
12-19 14:06:12.577  5202  5202 D ArticlesListFragment: Fragment 2 onResume()
12-19 14:06:12.578  5202  5202 D ArticlesListFragment: Fragment 2 onShow()
12-19 14:06:53.428  5202  5202 I ListPopupWindow: Could not find method setEpicenterBounds(Rect) on PopupWindow. Oh well.
12-19 14:06:53.442  5202  5202 W art     : Before Android 4.1, method int android.support.v7.widget.ListViewCompat.lookForSelectablePosition(int, boolean) would have incorrectly overridden the package-private method in android.widget.ListView
12-19 14:06:54.707  5202  5202 D ArticlesListFragment: Fragment 0 onPause()
12-19 14:06:54.707  5202  5202 D ArticlesListFragment: Fragment 1 onPause()
12-19 14:06:54.707  5202  5202 D ArticlesListFragment: Fragment 2 onPause()
12-19 14:06:54.769  5202  5202 D SettingsFragment: onStart() started
12-19 14:06:54.769  5202  5202 D SettingsFragment: resetChanges() started
12-19 14:06:54.883  5202  5228 D OpenGLRenderer: endAllStagingAnimators on 0x9dfc2000 (MenuPopupWindow$MenuDropDownListView) with handle 0x9e9f80d0
12-19 14:06:58.790  5202  5519 D TestConnectionTask: Detecting redirection for: https://framabag.org/u/username
12-19 14:07:00.191  5202  5519 D TestConnectionTask: URLs: 
12-19 14:07:00.191  5202  5519 D TestConnectionTask: https://framabag.org/u/username
12-19 14:07:00.191  5202  5519 I TestConnectionTask: Testing https://framabag.org/u/username
12-19 14:07:00.192  5202  5519 D WallabagService: getBodyFromHttpResponse() url=https://framabag.org/u/username/api/version
12-19 14:07:00.712  5202  5519 D WallabagService: getBodyFromHttpResponse() response.code is not OK: 404
12-19 14:07:00.712  5202  5519 D WallabagService: isWallabagVersion2() did not find Wallabag v2
12-19 14:07:00.712  5202  5519 D WallabagService: getBodyFromHttpResponse() url=https://framabag.org/u/username/?view=about
12-19 14:07:01.391  5202  5519 D WallabagService: isWallabagVersion1() found Wallabag v1
12-19 14:07:04.011  5202  5555 W OkHttp  : A connection to https://framabag.org/ was leaked. Did you forget to close a response body?
12-19 14:07:04.013  5202  5519 D TestConnectionTask: Connection test result: OK
12-19 14:07:04.045  5202  5202 D ConfigurationTestHelper: connectionTestOnSuccess() new url: https://framabag.org/u/username
12-19 14:07:04.060  5202  5592 D WallabagServiceEndpoint: getCredentials(): configPath=/?view=config credentialsPattern="\?feed&amp;type=home&amp;user_id=(\d+)&amp;token=([a-zA-Z0-9]+)"
12-19 14:07:04.060  5202  5592 D WallabagServiceEndpoint: getConfigRequest() url: https://framabag.org/u/username/?view=config
12-19 14:07:04.061  5202  5592 D WallabagServiceEndpointV1: executeRequestForResult(url: https://framabag.org/u/username/?view=config, checkResponse: true, autoRelogin: true) started
12-19 14:07:05.123  5202  5592 D WallabagServiceEndpointV1: executeRequestForResult() got response
12-19 14:07:05.125  5202  5592 D WallabagServiceEndpointV1: executeRequestForResult() response is login page
12-19 14:07:05.125  5202  5592 D WallabagServiceEndpointV1: executeRequestForResult() trying to re-login
12-19 14:07:05.411  5202  5592 D WallabagServiceEndpointV1: executeRequestForResult() re-login response is OK; re-executing request
12-19 14:07:06.632  5202  5592 D WallabagServiceEndpointV1: executeRequestForResult() finished; returning page body
12-19 14:07:06.633  5202  5592 D WallabagServiceEndpoint: credentials=FeedsCredentials{userID='9999', token='sihpoihiuhiug'}
12-19 14:07:07.439  5202  5605 I TestFeedsTask: Result: OK
12-19 14:07:07.460  5202  5202 D SettingsFragment: onConfigurationTestSuccess(https://framabag.org/u/username, 9999)
12-19 14:07:07.461  5202  5202 D SettingsFragment: onSharedPreferenceChanged(internal.configurationIsOk)
12-19 14:07:07.461  5202  5202 D SettingsFragment: onSharedPreferenceChanged(internal.configurationErrorShown)
12-19 14:07:07.571  5202  5228 V RenderScript: 0x9e005000 Launching thread(s), CPUs 4
12-19 14:07:11.037  5202  5202 W InputEventReceiver: Attempted to finish an input event but the input event receiver has already been disposed.
12-19 14:07:12.865  5202  5202 D ArticlesListActivity: checkLists() started
12-19 14:07:12.865  5202  5202 D ArticlesListActivity: checkLists() finished
12-19 14:07:12.866  5202  5202 D ServiceHelper: updateFeed() started
12-19 14:07:12.872  5202  5202 D ServiceHelper: updateFeed() finished
12-19 14:07:12.872  5202  5202 D ArticlesListFragment: Fragment 0 onResume()
12-19 14:07:12.872  5202  5202 D ArticlesListFragment: Fragment 0 onShow()
12-19 14:07:12.872  5202  5202 D ArticlesListFragment: Fragment 1 onResume()
12-19 14:07:12.872  5202  5202 D ArticlesListFragment: Fragment 1 onShow()
12-19 14:07:12.872  5202  5202 D ArticlesListFragment: Fragment 2 onResume()
12-19 14:07:12.872  5202  5202 D ArticlesListFragment: Fragment 2 onShow()
12-19 14:07:12.893  5202  5202 D BGService: BGService() created
12-19 14:07:12.894  5202  5629 D BGService: onHandleIntent() started
12-19 14:07:12.910  5202  5629 D EventProcessor: onUpdateFeedsStartedEvent() started
12-19 14:07:12.920  5202  5629 D BGService: updateFeed(null, null) started
12-19 14:07:12.920  5202  5629 D DbConnection: using existing db session
12-19 14:07:12.921  5202  5629 I FeedUpdater: updateAllFeeds() started
12-19 14:07:12.921  5202  5629 D DbConnection: using existing db session
12-19 14:07:12.921  5202  5629 D FeedUpdater: updateAllFeeds() deleting old articles
12-19 14:07:12.940  5202  5629 D FeedUpdater: updateAllFeeds() updating Main feed
12-19 14:07:12.940  5202  5629 D FeedUpdater: updateByFeed() started
12-19 14:07:12.962  5202  5202 D ArticlesListActivity: Got UpdateFeedsStartedEvent
12-19 14:07:13.185  5202  5202 D SettingsFragment: onStop() started
12-19 14:07:13.185  5202  5202 D SettingsFragment: applyChanges() started
12-19 14:07:14.824  5202  5629 W BGService: updateFeed() fr.gaulupeau.apps.Poche.network.exceptions.IncorrectConfigurationException
12-19 14:07:14.824  5202  5629 W BGService: fr.gaulupeau.apps.Poche.network.exceptions.IncorrectConfigurationException: Request unsuccessful; response code: 500, response message: Internal Server Error
12-19 14:07:14.824  5202  5629 W BGService:     at fr.gaulupeau.apps.Poche.network.FeedUpdater.getInputStream(FeedUpdater.java:203)
12-19 14:07:14.824  5202  5629 W BGService:     at fr.gaulupeau.apps.Poche.network.FeedUpdater.updateByFeed(FeedUpdater.java:165)
12-19 14:07:14.824  5202  5629 W BGService:     at fr.gaulupeau.apps.Poche.network.FeedUpdater.updateAllFeeds(FeedUpdater.java:122)
12-19 14:07:14.824  5202  5629 W BGService:     at fr.gaulupeau.apps.Poche.network.FeedUpdater.update(FeedUpdater.java:96)
12-19 14:07:14.824  5202  5629 W BGService:     at fr.gaulupeau.apps.Poche.service.BGService.updateFeed(BGService.java:363)
12-19 14:07:14.824  5202  5629 W BGService:     at fr.gaulupeau.apps.Poche.service.BGService.onHandleIntent(BGService.java:99)
12-19 14:07:14.824  5202  5629 W BGService:     at android.app.IntentService$ServiceHandler.handleMessage(IntentService.java:66)
12-19 14:07:14.824  5202  5629 W BGService:     at android.os.Handler.dispatchMessage(Handler.java:102)
12-19 14:07:14.824  5202  5629 W BGService:     at android.os.Looper.loop(Looper.java:148)
12-19 14:07:14.824  5202  5629 W BGService:     at android.os.HandlerThread.run(HandlerThread.java:61)
12-19 14:07:14.833  5202  5629 D BGService: updateFeed() finished
12-19 14:07:14.834  5202  5629 D EventProcessor: onUpdateFeedsFinishedEvent() started
12-19 14:07:14.840  5202  5629 D EventProcessor: onActionResultEvent() started
12-19 14:07:14.840  5202  5629 D EventProcessor: onActionResultEvent() action: UpdateFeed
12-19 14:07:14.841  5202  5629 D EventProcessor: onActionResultEvent() result is not success; errorType: IncorrectConfiguration
12-19 14:07:14.841  5202  5629 D EventProcessor: onActionResultEvent() result message: null
12-19 14:07:14.848  5202  5629 D EventProcessor: onActionResultEvent() notification is not null; showing it
12-19 14:07:14.857  5202  5202 D ArticlesListActivity: Got UpdateFeedsFinishedEvent
12-19 14:07:14.869  5202  5629 D BGService: onHandleIntent() finished
12-19 14:07:29.214  5202  5202 D ArticlesListFragment: Fragment 0 onPause()
12-19 14:07:29.214  5202  5202 D ArticlesListFragment: Fragment 1 onPause()
12-19 14:07:29.214  5202  5202 D ArticlesListFragment: Fragment 2 onPause()
12-19 14:07:35.388  5202  5202 D ArticlesListActivity: checkLists() started
12-19 14:07:35.388  5202  5202 D ArticlesListActivity: checkLists() finished
12-19 14:07:35.410  5202  5202 D ArticlesListFragment: Fragment 0 onResume()
12-19 14:07:35.410  5202  5202 D ArticlesListFragment: Fragment 0 onShow()
12-19 14:07:35.410  5202  5202 D ArticlesListFragment: Fragment 1 onResume()
12-19 14:07:35.410  5202  5202 D ArticlesListFragment: Fragment 1 onShow()
12-19 14:07:35.410  5202  5202 D ArticlesListFragment: Fragment 2 onResume()
12-19 14:07:35.410  5202  5202 D ArticlesListFragment: Fragment 2 onShow()
12-19 14:07:35.417  5202  5228 I Adreno-EGL: <qeglDrvAPI_eglInitialize:379>: QUALCOMM Build: 10/21/15, 369a2ea, I96aee987eb
12-19 14:07:35.417  5202  5228 I OpenGLRenderer: Initialized EGL, version 1.4
12-19 14:07:45.393  5202  5228 V RenderScript: 0x9e00e000 Launching thread(s), CPUs 4
```

**Comment by di72nn** [comment]
Thank you for all your efforts.
Now we can see the cause of the problem: the server fails to generate the "unread feed": `500 Internal Server Error` (and the app fails to handle the error gracefully). You can check it in your browser by clicking on "Unread feed" in "Config - Feeds".

Similar problem was discussed in #97. If I recall correctly, it can be caused either by a specific article or just by the large amount of articles. If you think you have not so much articles, we can summon framabag admins and try to get some help tracking down the problem article.

Anyway, I would recommend you to migrate to wallabag v2. There is an official v2 instance: https://wallabag.it, but it requires subscription fees.

**Comment by akwala** [comment]
Yep, I have 2000+. I guess the thing to do is migrate to v2.

Since #97 is open, I am closing this one.

Thanks.
